checks/check_status.py

```diff
@@ -757,7 +757,7 @@ class ForwarderStatus(AgentStatus):
     NAME = 'Forwarder'
 
     def __init__(self, queue_length=0, queue_size=0, flush_count=0, transactions_received=0,
-                 transactions_flushed=0):
+            transactions_flushed=0, too_big_count=0):
```

Comment:
I don't think so. I'll change it back. I think it was just what my autoindent did.